lib/inspec/resources/sys_info.rb

```diff
@@ -26,5 +26,29 @@ def hostname
         skip_resource "The `sys_info.hostname` resource is not supported on your OS yet."
       end
     end
+
+    # returns the Manufacturer of the local system
+    def manufacturer
+      os = inspec.os
+      if os.linux? || os.darwin?
+        inspec.command("dmidecode | grep 'Vendor :'").split(": ").last.chomp
+      elsif os.windows?
+        inspec.powershell("Get-CimInstance -ClassName Win32_ComputerSystem | Select Manufacturer -ExpandProperty Manufacturer").stdout.chomp
+      else
+        skip_resource "The `sys_info.model` resource is not supported on your OS yet."
+      end
+    end
+
+    # returns the ServerModel of the local system
+    def model
+      os = inspec.os
+      if os.linux? || os.darwin?
+        inspec.command("dmidecode | grep 'Product Name: '").split(": ").last.chomp
```

Comment:
Likewise, no dmidecode on MacOS - we should let darwin fall thru to unsupported or find an alternative implementation

Reply:
I have no OSX at hand, but according to https://serverfault.com/questions/112711/how-can-i-get-cpu-count-and-total-ram-from-the-os-x-command-line/397809#397809, `system_profiler` should also work. Added it in the latest commit.

Reply:
Changed it from `system_profiler` to `sysctl` for faster execution time.